src/PAModel/Serializers/MsAppSerializer.cs

```diff
@@ -253,14 +253,14 @@ public static CanvasDocument Load(Stream streamToMsapp, ErrorContainer errors)
                 // Normalize logo filename. 
                 app.TranformLogoOnLoad();
 
-                if (app._properties.LibraryDependencies != null)
+                if (!string.IsNullOrEmpty(app._properties.LibraryDependencies))
```

Comment:
>string.IsNullOrEmpty [](http://example.com/codeflow?start=21&length=20)

The one danger to be aware of here is that this won't round-trip: we'll read in empty string and write out null. 
That will result in a checksum mismatch. 
But that's probalby fine in this case (since this is just extra hardening and the server should alway write out null).

Reply:
the problem is if we don't check for empty string the code would throw an exception in the very next line where it would try to deserialize empty string as json, yes we can probably do special handling for empty strings vs null but haven't seen any such instance yet.